src/app/index.test.js

```diff
@@ -88,55 +112,59 @@ describe('generator:app', () => {
     })
   })
   describe('LICENSE.md', () => {
     it('contains the current year and author name', async () => {
       Date.prototype.getUTCFullYear = jest.fn(() => 2017)
-      await helpers.run(__dirname).withPrompts({
-        pluginName: 'danger-plugin-fun-time',
-        description: 'Danger plugin that tells you to have a fun time',
-        authorName: 'Macklin Underdown',
-        authorEmail: 'email@example.com',
-      })
+      await helpers.run(__dirname).withPrompts(exampleSettings)
       expect(readFile('LICENSE.md')).toMatchSnapshot()
     })
   })
   describe('package.json', () => {
-    it('fills package.json with correct information', async () => {
+    it('fills package.json with correct JS information', async () => {
       await helpers.run(__dirname).withPrompts({
-        pluginName: 'danger-plugin-fun-time',
-        description: 'Danger plugin that tells you to have a fun time',
-        authorName: 'Macklin Underdown',
-        authorEmail: 'email@example.com',
-        githubUsername: 'macklinu',
-        keywords: ['fun', 'time'],
+        ...exampleSettings,
+        useTypeScript: false,
       })
       const pkg = readFile('package.json')
       expect(pkg).toMatchSnapshot()
     })
-    it('uses git user.name as default name prompt value', async () => {
+    it('fills package.json with correct TS information', async () => {
       await helpers.run(__dirname).withPrompts({
-        pluginName: 'danger-plugin-fun-time',
-        description: 'Danger plugin that tells you to have a fun time',
+        ...exampleSettings,
+        useTypeScript: true,
       })
       const pkg = readFile('package.json')
+      expect(pkg).toMatchSnapshot()
+    })
+    it('uses git user.name as default name prompt value', async () => {
+      await helpers.run(__dirname).withPrompts(exampleSettings)
+      const pkg = readFile('package.json')
       expect(pkg.author.name).toEqual('Git Name')
     })
     it('uses git user.email as default email prompt value', async () => {
+      await helpers.run(__dirname).withPrompts(exampleSettings)
+      const pkg = readFile('package.json')
+      expect(pkg.author.email).toEqual('git@git.com')
+    })
+    it('uses tsc for TypeScript builds', async () => {
       await helpers.run(__dirname).withPrompts({
-        pluginName: 'danger-plugin-fun-time',
-        description: 'Danger plugin that tells you to have a fun time',
+        ...exampleSettings,
+        useTypeScript: true,
       })
       const pkg = readFile('package.json')
-      expect(pkg.author.email).toEqual('git@git.com')
+      expect(pkg.scripts.build).toContain('tsc')
+    })
+    it('uses babel for JavaScript builds', async () => {
+      await helpers.run(__dirname).withPrompts({
+        ...exampleSettings,
+        useTypeScript: false,
+      })
+      const pkg = readFile('package.json')
+      expect(pkg.scripts.build).toContain('babel')
     })
   })
```

Comment:
there are, like, no visual spaces in this entire file?!? 
How can you live with yourself? :O